templates/focus-ios.xliff

```diff
@@ -557,6 +658,18 @@
         <source>Example: searchengineexample.com/search/?q=%s</source>
         <note>Text displayed as an example of the template to add a search engine.</note>
       </trans-unit>
+      <trans-unit id="share">
+        <source>Share</source>
+        <note>Text for a share button</note>
+      </trans-unit>
+      <trans-unit id="share.trackerStatsLabel">
+        <source>%@ trackers blocked so far</source>
+        <note>Text used when the user shares their trackers blocked stats</note>
```

Comment:
It's always good to say what a placeholder stands for so people know where to place it in the string once they translate (even though here it's easy to guess for me, some people have no clue). So like "%@ will be replaced by blablabla"

Reply:
This won't work. "X trackers" requires a proper plural form, which we currently don't support in XLIFF.
https://bugzilla.mozilla.org/show_bug.cgi?id=1181777

This should be something like `Trackers blocked: %@`